src/test/java/com/hocs/test/pages/homepage/Homepage.java

```diff
@@ -114,6 +159,31 @@ public void selectCentralDraftingTeam() {
         centralDraftingTeam.click();
     }
 
+    public void selectPoliceWorkforceProfessionalismTeam() { policeWorkforceProfessionalismUnit.click(); }
```

Comment:
Few formatting discrepancies here. We generally have the closing bracket on a new line.